components/self_hosted_purchase_modal/consequences.tsx

```diff
@@ -44,6 +46,28 @@ export default function Consequences(props: Props) {
             }}
         />);
 
+    const licenseAgreement = (
```

Comment:
This might be the rare case where we don't want **any** internationalization. Other places that deal with licensing wording, we deliberately do not translate, and have comments like the following:

```
components/admin_console/license_settings/license_settings.tsx
292:            // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
313:            // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
336:            // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.

components/admin_console/license_settings/modals/ee_license_modal.tsx
40:    // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.

components/admin_console/license_settings/enterprise_edition/enterprise_edition_left_panel.tsx
190:    // Note: DO NOT LOCALISE THESE STRINGS. Legally we can not since the license is in English.
```

@johndavidlugtu would it be appropriate to show this license agreement text in English regardless of the user's preferred language?

Reply:
Yes, that's right @johndavidlugtu

Reply:
Do we internationalize license agreement text elsewhere? (e.g. trial start)

Reply:
Yes, we do elsewhere, but those might not have taken into account Nathaniel's concern.

Reply:
@ConorMacpherson @neallred we're okay to localize this. Let's just make sure it's consistent across all mentions of legal terms (e.g. trial, purchase screens, etc)